src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1593,6 +1593,87 @@ public void MainWindowHandle_GetWithGui_ShouldRefresh_Windows()
             }
         }
 
+        [Fact]
+        [OuterLoop]
+        [Trait(XunitConstants.Category, XunitConstants.IgnoreForCI)] // Pops UI
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void MainWindowTitle_GetWithGui_ShouldRefresh_Windows()
+        {
+            const string ExePath = "notepad.exe";
+            Assert.True(IsProgramInstalled(ExePath));
+
+            using (Process process = Process.Start(ExePath))
+            {
+                try
+                {
+                    Assert.Equal(string.Empty, process.MainWindowTitle);
```

Comment:
Why do we expect MainWindowTitle to start as empty and then be non-empty after a Refresh?

Reply:
I see this behaviour with the test program that I've written:
https://github.com/JeroenOortwijn/MainWindowTitle-test

With .Net Framework 4.8, the first `Process.MainWindowTitle` call (immediately after Notepad is started) returns an empty string.
As soon as the window is visible, it returns the contents of the window title.

Due to issue #36768, `Process.MainWindowTitle` keeps returning an empty string on .Net Core 3.1 and .Net 5.

Reply:
Ok. Thanks.